### PR TITLE
Feature/YLP-1466 create alarm card

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

File: packages/yourloops/lib/units/utils.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-export const MMOLL_PER_MGDL = 0.0555;
+export const MGDL_PER_MMOL = 18;
```
Comment (CedricPerotto) [inline packages/yourloops/lib/units/utils.tsx:28]: As described here (it should be the same rule in all DBL products):
https://docs.google.com/spreadsheets/d/1P7b41oHoV73t5OgXFe22vo1Wso38u9ixgk3v0O81nKY/edit#gid=620459482

```suggestion
export const MGDL_PER_MMOL = 18.01577;
```

**Review by CedricPerotto — CHANGES_REQUESTED** [review]
Uncomplete review... just the unit conversion.

File: packages/yourloops/components/alarm/alarms-content.tsx [hunk 2]
```diff
@@ -88,6 +91,8 @@ const useStyles = makeStyles((theme: Theme) => ({
 export interface TeamAlarmsContentProps {
```
Comment (Nas84) [inline packages/yourloops/components/alarm/alarms-content.tsx:91]: What do you think to rename as AlarmsContentConfigurationProps ?

File: packages/yourloops/components/alarm/alarms-content.tsx [hunk 3]
```diff
@@ -103,10 +108,10 @@ export const MAX_VERY_LOW_BG = 90;
 export const MIN_LOW_BG = 50;
 export const MAX_LOW_BG = 100;
 export const PERCENTAGES = [...new Array(21)]
-  .map((_each, index) => `${index * 5}%`);
+  .map((_each, index) => `${index * 5}%`).slice(1, 21);
 
-function TeamAlarmsContent(props: TeamAlarmsContentProps): JSX.Element {
-  const { monitoring, saveInProgress, onSave } = props;
+function AlarmsContent(props: TeamAlarmsContentProps): JSX.Element {
```
Comment (Nas84) [inline packages/yourloops/components/alarm/alarms-content.tsx:113]: What do you think to rename as AlarmsContentConfiguration ?

**Review by Nas84 — CHANGES_REQUESTED** [review]

**Review by Nas84 — APPROVED** [review]
LGTM

**Review by CedricPerotto — APPROVED** [review]

**Pull request merged**